include/aspect/material_model/rheology/drucker_prager.h

```diff
@@ -107,6 +112,7 @@ namespace aspect
           double
           compute_yield_stress (const double cohesion,
                                 const double angle_internal_friction,
+                                const double yield_stress_prefactor,
```

Comment:
I think there should actually be a way around adding this parameter here, in order to not break backwards compatibility. Hopefully the solution will be apparent later on.

Reply:
Yes, please remove this line